From 0dc8154f087a0b2b9b2e53351592d16e3c278786 Mon Sep 17 00:00:00 2001
From: Alexandru Csete <oz9aec@gmail.com>
Date: Sat, 1 Oct 2016 12:57:17 +0200
Subject: [PATCH 77/96] rfspace: Wait 10 ms before sending queries to device.

This is necessary to esatablish a working connection to the RFSpace
CloudIQ. Without this delay the radio will not be ready and we never
receive any response to the queries and the radio will close the
connection after 5 seconds.
---
 lib/rfspace/rfspace_source_c.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/rfspace/rfspace_source_c.cc b/lib/rfspace/rfspace_source_c.cc
index 93d645d..f9c587d 100644
--- a/lib/rfspace/rfspace_source_c.cc
+++ b/lib/rfspace/rfspace_source_c.cc
@@ -344,6 +344,7 @@ rfspace_source_c::rfspace_source_c (const std::string &args)
   }
 
   /* request & print device information */
+  usleep(10000);
 
   std::vector< unsigned char > response;
 
-- 
2.11.0

